unstructured/metrics/element_type.py

```diff
@@ -1,10 +1,23 @@
+from __future__ import annotations
+
 import json
-from typing import Dict, Optional, Tuple, Union
+
+from typing_extensions import TypeAlias
+
+FrequencyDict: TypeAlias = "dict[tuple[str, int | None], int]"
+"""Like:
+    {
+        ("ListItem", 0): 2,
+        ("NarrativeText", None): 2,
+        ("Title", 0): 5,
+        ("UncategorizedText", None): 6,
+    }
```

Comment:
Why is there an integer in the key? Wouldn't you just need the string for the key and the int/frequency as the value?

Reply:
I didn't invent this data structure, but from what I gather, the `int | None` member of the tuple is to record the `category_level` value for the element. So like how many top-level headings do we have separate from how many second and third-level headings. Something in the metrics wants to make sure we detect the right level for headings and list items is my take.

Reply:
Btw, the reason I had to touch this was because of a "bug fix" or at least an improvement along the way.

The file `fake-email.txt` was previously identified as TXT and partitioned quite differently because all the transport headers and so forth were included, as garbage text I'm sure.

But the refactored version correctly identified it as EML and partitions it as a user would intend. That resulted in fewer elements, partly because it's a very short example email.

Reply:
Okay. Didn't realize this type was used elsewhere.